index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>Apple</title>
+	<script src="https://kit.fontawesome.com/1cfd58b03e.js"></script>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+	<main class="main-container">
+		<section class="main-section">
+			<div><nav class="navbar">
+			<ul id="global-nav">
```

Comment:
Micorverse recommend using floats or display: flex for this section

Reply:
used display:flex in the navbar class